src/lib/Layout/PanelSideBarLayout/PanelSideBar/PanelSideBarItem.tsx

```diff
@@ -31,40 +38,42 @@ const PanelSideBarItem = (props: PanelSideBarItemProps) => {
         className={classNames({ "menu-open": isOpen, active: item.children?.find((s) => s.active) || item.active })}
         style={{ paddingLeft: depth ? `${depth + 1}rem` : undefined }}
       >
-        {hasitem ? (
-          <div className={classNames("d-flex flex-row", { "justify-content-between": item.collapseIconOnly })}>
-            {item.collapseIconOnly && (
+        <div ref={scollToActiveItemRef}>
+          {hasitem ? (
+            <div className={classNames("d-flex flex-row", { "justify-content-between": item.collapseIconOnly })}>
+              {item.collapseIconOnly && (
+                <LinkRenderer item={item}>
+                  <span className="nav-link">
+                    {item.icon && <FontAwesomeIcon icon={item.icon} className="me-2" />}
+                    {item.title}
+                  </span>
+                </LinkRenderer>
+              )}
+
+              <a
+                role="button"
+                className={classNames("nav-link", { "w-100": !item.collapseIconOnly }, { "dropdown-toggle": hasitem })}
+                onClick={() => setIsOpen(!isOpen)}
+              >
+                {!item.collapseIconOnly && (
+                  <span>
+                    {item.icon && <FontAwesomeIcon className="me-2" icon={item.icon} />}
+                    {item.title}
+                  </span>
+                )}
+              </a>
+            </div>
+          ) : (
+            <>
               <LinkRenderer item={item}>
```

Comment:
This works only if the item has a children. I think this is not what we expect. FFS has a long menu also without opened list. So I would extend the logic